app/common/azure_blob_storage_handler.py

```diff
@@ -0,0 +1,50 @@
+import os
+import uuid
+
+from azure.storage.blob import BlobServiceClient, ContentSettings
+from sentry_sdk import capture_exception
+
+
+def get_container_or_create_if_not_exist(name="default"):
```

Comment:
Hva med et kortere og mer konsist metodenavn?
```suggestion
def get_or_create_container(name="default"):
```